tlsn/examples/src/lib.rs

```diff
@@ -155,3 +159,9 @@ pub async fn request_notarization(
         notarization_response.session_id,
     )
 }
+
+pub fn print_with_time(input: impl ToString, time: Instant) {
```

Comment:
can use the tracing crate for logging so verbosity level can be controlled easier :)